template/.github/workflows/deploy_preview.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Preview to Netlify
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  deploy-preview:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Should we add this GitHub actions as the first step:
https://github.com/styfle/cancel-workflow-action

Then it can stop the previous run when we run another one 🤔

Reply:
50c2b66 ✅